File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int start = 0;
        int value = nums[start];
        int cur = 1;
        while(cur < nums.length){
            if(nums[cur] != value){
                value = nums[cur];
                start++;
                nums[start] = value;
            }
            cur++;
        }
        return start + 1;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        start = 0
        set = {}
        for i in range(1, len(nums)): 
            if nums[i] == nums[start]:
                continue
            else:
                start += 1
                nums[start] = nums[i]
        return start + 1
        